back/src/chats/chats.resolver.ts

```diff
@@ -0,0 +1,53 @@
+import { Resolver, Query, Mutation, Args, Int } from '@nestjs/graphql';
+import { ChatsService } from './chats.service';
+import { Chat } from './entities/chat.entity';
+import { CreateChatInput } from './dto/create-chat.input';
+import { UpdateChatInput } from './dto/update-chat.input';
+
+@Resolver(() => Chat)
+export class ChatsResolver {
+  constructor(private readonly chatsService: ChatsService) {}
+
+  @Mutation(() => Chat)
+  createChat(@Args('createChatInput') createChatInput: CreateChatInput) {
+    return this.chatsService.create(createChatInput);
+  }
+
+  @Query(() => [Chat], { name: 'chats' })
+  findAll() {
+    return this.chatsService.findAll();
+  }
+
+  @Query(() => Chat, { name: 'chat' })
+  findOne(@Args('uuid', { type: () => String }) uuid: string) {
+    return this.chatsService.findOne(uuid);
+  }
+
+  @Mutation(() => Chat)
+  updateChat(@Args('updateChatInput') updateChatInput: UpdateChatInput) {
+    return this.chatsService.update(updateChatInput.uuid, updateChatInput);
+  }
+
+  @Mutation(() => Chat)
+  removeChat(@Args('uuid', { type: () => String }) uuid: string) {
+    return this.chatsService.remove(uuid);
+  }
+
+  @Query(() => Int)
+  getChatCount(
+    @Args('type', { type: () => String, nullable: true }) type: string,
+    @Args('userID', { type: () => String, nullable: true }) userID: string,
+  ) {
+    return this.chatsService.getCount({ type, userID });
+  }
+
+  @Query(() => [Chat], { name: 'aliveChats' })
+  findAliveMyChats(
```

Comment:
```suggestion
  findMyAliveChats(
```
표현이 더 자연스러운 것 같아 제안합니다!

Reply:
findAliveChats로 수정해야 했는데 이 부분을 놓쳐버렸네요.. 수정해 놓도록 하겠습니다